boards/arm/stm32h7/nucleo-h743zi2/include/board.h

```diff
@@ -390,8 +390,14 @@
 #define GPIO_TIM1_CH2OUT  GPIO_TIM1_CH2OUT_2  /* PE11 - D5 */
 #define GPIO_TIM1_CH2NOUT GPIO_TIM1_CH2NOUT_3 /* PE10 - D40 */
 
+#define GPIO_TIM1_CH1IN GPIO_TIM1_CH1IN_2       /* PE9    */
```

Comment:
why not merge into the second patch

Reply:
You're right! 
I'll remove the OK definition